evalml/pipelines/utils.py

```diff
@@ -151,6 +151,7 @@ def make_pipeline_from_components(component_instances, problem_type, custom_name
         problem_type (str or ProblemTypes): problem type for the pipeline to generate
         custom_name (string): a name for the new pipeline
         random_state (int or np.random.RandomState): Random state used to intialize the pipeline.
+        kwargs: Additional arguments that can be passed to the created pipeline
```

Comment:
Is this necessary? I don't see it in the signature.

Reply:
Ah, good catch! I forgot to remove this.